### PR TITLE
[Front-End] User List

### PR DESCRIPTION
**Pull request by nex-pixel**
## Feature Description/Implementation
User list is done with sorting enabled for both columns in both pagination and non pagination format.
Fixed some errors such as the delete button not being disabled. 
Also corrected some fields that were labeled badly.

### CODE REVIEW

File: frontend/src/apis/users/UserAPI.tsx
Comment (caseyszilagyi) [inline frontend/src/apis/users/UserAPI.tsx:75]: Since you updated everything else to have no pagination, should probably change this one too

File: frontend/src/pages/users/UserDetail.tsx
Comment (caseyszilagyi) [inline frontend/src/pages/users/UserDetail.tsx:54]: This looks good, but there are a couple other requirements that I just wanted to ask about to make sure I'm up to date

1. "The admin account is special and cannot lose adminstrator privilege." I think this would just be disabling the toggle for the admin account?
2. The admin account username cannot be changed

**Review by caseyszilagyi — CHANGES_REQUESTED** [review]

File: frontend/src/apis/users/UserAPI.tsx
Comment (caseyszilagyi) [reply frontend/src/apis/users/UserAPI.tsx:75]: If you want, feel free to wait until the no pagination changes are merged into main, or just merge this branch with that one

File: frontend/src/pages/users/UserDetail.tsx
Comment (nex-pixel) [reply frontend/src/pages/users/UserDetail.tsx:54]: Yes the toggle is disabled. Right above this is a method I had made that makes the page disabled for editing.
Currently, no user that uses the website can change their name. I didn't see the requirement as to if they can. So admin's name can't be changed.

Comment (caseyszilagyi) [reply frontend/src/pages/users/UserDetail.tsx]: Didn't realize user's couldn't change their name, that's my bad. And sounds good on the admin toggle, looks like everything is covered.

**Review by caseyszilagyi — APPROVED** [review]

**Pull request merged**
